oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStore.java

```diff
@@ -571,6 +571,8 @@ public Long apply(@Nullable String input) {
 
     private final Feature cancelInvalidationFeature;
 
+    private final Feature noChildOrderCleanupFeature;
```

Comment:
```suggestion
    private final Feature childOrderCleanupFeature;
```

Reply:
Well, I just changed it from childOrderCleanupFeature to noChildOrderCleanupFeature, because that's what it is since we changed the default behavior.

Reply:
Irrespective of the default value, I think the name `childOrderCleanupFeature` seems more intuitive.

Reply:
I think it would be misleading. The name childOrderCleanupFeature suggests that enabling the feature would enable child order cleanup.